### DISCUSSION TITLE
Could we have some way for dashboard URLs to never include slugs?

### DISCUSSION THREAD

**Discussion by peterbourgon** [grafana/grafana#47237] [Dashboards, Panels & Visualizations]
I have a provisioning workflow which appends version and updated-at date to dashboard titles when they're modified. This is useful metadata, and the title seems like the right place for it. But because titles are slug-ified and included in the URL by default, this means that those URLs aren't actually useful — once a dashboard's title changes, all existing e.g. bookmarks break.

I believe that given `/d/{uid}/{slug}` the `/d/{uid}` already uniquely identifies the dashboard, correct? Could we have some way to tell Grafana that a dashboard's canonical URL is simply `/d/{uid}` and never include the auto-generated title slug?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
